clang/include/clang/Basic/Attr.td

```diff
@@ -3341,7 +3341,7 @@ def RequiresCapability : InheritableAttr {
   let TemplateDependent = 1;
   let ParseArgumentsAsUnevaluated = 1;
   let InheritEvenIfAlreadyPresent = 1;
-  let Subjects = SubjectList<[Function]>;
+  let Subjects = SubjectList<[Function, FunctionPointer]>;
   let Accessors = [Accessor<"isShared", [Clang<"requires_shared_capability", 0>,
                                          Clang<"shared_locks_required", 0>]>];
   let Documentation = [Undocumented];
```

Comment:
I think now is a good time to insist we document this :)  It would at least make review easier...

Reply:
FWIW: https://clang.llvm.org/docs/ThreadSafetyAnalysis.html

So perhaps we could give some very high-level documentation but link to there? (At the very least, hopefully this helps you with review efforts.)

Reply:
Oooh, yeah, i think that would be enough if this/others linked to it.  I went to our attributes page to figure out what this does, and saw "not documented".  Just a 2-sentence summary + "This is part of Thread Safety Analysis, as documented here" would be great!  

Also, jsut figured out what "TSA" refers to here...